server/text_generation_server/layers/attention/cuda.py

```diff
@@ -254,9 +290,11 @@ def attention(
         softcap=None,
     ):
         if window_size_left != -1:
-            raise NotImplementedError(
-                "window_size_left is only available with flash attn v2"
+            warnings.warn(
```

Comment:
Can you also not change that.

It's important to hard crash and not silently ignored.

Reply:
updated in the latest commit to still throw when an invalid value is passed